src/main/scala/viper/gobra/ast/internal/utility/GobraStrategy.scala

```diff
@@ -40,7 +40,7 @@ object GobraStrategy {
       case (_: Seqn, Seq(stmts: Vector[Stmt@unchecked])) => Seqn(stmts)(meta)
       case (_: Label, Seq(label: LabelProxy)) => Label(label)(meta)
       case (_: If, Seq(cond: Expr, thn: Stmt, els: Stmt)) => If(cond, thn, els)(meta)
-      case (_: While, Seq(cond: Expr, invs: Vector[Assertion@unchecked], terminationMeasure: Option[TerminationMeasure@unchecked], body: Stmt)) => While(cond, invs, terminationMeasure, body)(meta)
+      case (_: While, Seq(cond: Expr, invs: Vector[Assertion@unchecked], terminationMeasure: Option[TerminationMeasure@unchecked], body: Stmt, label: Option[String])) => While(cond, invs, terminationMeasure, body, label)(meta)
```

Comment:
If `label` is kept as a filed, its type should be `Option[String@unchecked]` to avoid compiler warning